File: help/pt_pt/exercicios_da_ficha_11/Ficha11_016/Ficha11_016.py
# Programador           -


#::::::::::::::::::::::::::::::::::::::::::::::::::
#User Defined Functions Code
#::::::::::::::::::::::::::::::::::::::::::::::::::
def linhaCopa(sp, chars, txt):
    for i in range(0, sp, 1 ):
        print( " " , end="")
    for i in range(0, chars, 1 ):
        print(txt  , end="")
    print()

#::::::::::::::::::::::::::::::::::::::::::::::::::
def copa(alt, txt):
    for i in range(1, alt + 1, 1 ):
        sp = alt - i 
        chrs1 = i * 2 - 1 
        linhaCopa(sp , chrs1 , txt )

if __name__ == "__main__":
    alt =  int(input("Altura   : "))
    char =  str(input("Caractér : "))
    copa(alt , char )


#Python 3  - Automatic translated by Algorithmi 22.05
#(c) Ant0nio M@nso 	quarta, 2022 out 12 20:15

File: help/pt_pt/exercicios_da_ficha_11/Ficha11_016/Ficha11_016.js
// Programador           -

//Main Function
function main() {
    document.write("Altura   : " );// print text
    alt = parseInt(prompt("Altura   : " ,"0") );
    
    document.write(alt + "\n");// print input 
    document.write("Caractér : " );// print text
    char = prompt("Caractér : " ,"");
    
    document.write(char + "\n");// print input 
    copa(alt , char ) ;
}

//::::::::::::::::::::::::::::::::::::::::::::::::::
//User Defined Functions Code
//::::::::::::::::::::::::::::::::::::::::::::::::::
function linhaCopa(sp, chars, txt) {
    for( var i = 0; i < sp; i = i + 1 ) {
        document.write(" " );
    }
    for( var i = 0; i < chars; i = i + 1 ) {
        document.write(txt );
    }
    document.write("\n" );
}
//::::::::::::::::::::::::::::::::::::::::::::::::::
function copa(alt, txt) {
    for( var i = 1; i < alt + 1; i = i + 1 ) {
        var sp = alt - i ;
        var chrs1 = i * 2 - 1 ;
        linhaCopa(sp , chrs1 , txt ) ;
    }
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 20:15
